<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Last.fm Data Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/logo/logo.png</normaloff>:/logo/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="3">
     <layout class="QGridLayout" name="gridLayout_4" rowstretch="0" columnstretch="0,0">
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QStackedWidget" name="topTables">
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="albumsPage">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0" colspan="2">
           <widget class="QTableWidget" name="topAlbumsTableWidget">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>4</number>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderStretchLastSection">
             <bool>false</bool>
            </attribute>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QPushButton" name="limitButton">
            <property name="text">
             <string>Limit</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="songsPage">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableWidget" name="topSongsTableWidget">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="artistsPage">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QTableWidget" name="topArtistsTableWidget">
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustToContents</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionAdd_User"/>
    <addaction name="actionChange_User"/>
    <addaction name="actionDelete_User"/>
    <addaction name="actionUpdate_Data"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Collage</string>
    </property>
    <addaction name="actionGenerate_Collage"/>
   </widget>
   <widget class="QMenu" name="menuTables">
    <property name="title">
     <string>Tables</string>
    </property>
    <addaction name="actionTop_Albums"/>
    <addaction name="actionTop_Tracks"/>
    <addaction name="actionTop_Artists"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUser"/>
   <addaction name="menuTables"/>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad_Database">
   <property name="text">
    <string>Load Database</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionAdd_User">
   <property name="text">
    <string>Add User</string>
   </property>
  </action>
  <action name="actionChange_User">
   <property name="text">
    <string>Change User</string>
   </property>
  </action>
  <action name="actionDelete_User">
   <property name="text">
    <string>Delete User</string>
   </property>
  </action>
  <action name="actionUpdate_Data">
   <property name="text">
    <string>Update Data</string>
   </property>
  </action>
  <action name="actionGenerate_Collage">
   <property name="text">
    <string>Generate Collage</string>
   </property>
  </action>
  <action name="actionTop_Albums">
   <property name="text">
    <string>Top Albums</string>
   </property>
  </action>
  <action name="actionTop_Tracks">
   <property name="text">
    <string>Top Tracks</string>
   </property>
  </action>
  <action name="actionTop_Artists">
   <property name="text">
    <string>Top Artists</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
